android/java/org/chromium/chrome/browser/vpn/utils/BraveVpnProfileUtils.java

```diff
@@ -63,7 +63,7 @@ public boolean isVPNRunning(Context context) {
     }
 
     public void startVpn(Context context) {
-        BraveVpnNativeWorker.getInstance().reportForegroundP3A();
+        // BraveVpnNativeWorker.getInstance().reportForegroundP3A();
```

Comment:
@DJAndries found an issue here. it's calling the native function and when we try to change the server, it crashes. so for now i have commented out the code.